### PR TITLE
Update local ephemeral storage feature to beta

### PR DESCRIPTION
**Pull request by jingxu97**
update local ephemeral storage feature to beta

### CODE REVIEW

**Comment by zhangxiaoyu-zidif** [comment]
could you change this PR to merge to release-1.0. =) 
and suggest to delete 'and above'.

File: docs/concepts/configuration/manage-compute-resources-container.md [hunk 1]
```diff
@@ -306,7 +306,7 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 You can see that the Container was terminated because of `reason:OOM Killed`,
 where `OOM` stands for Out Of Memory.
 
-## Local ephemeral storage (alpha feature)
+## Local ephemeral storage (beta feature for version 1.10.0 and above)
```
Comment (heckj) [inline docs/concepts/configuration/manage-compute-resources-container.md]: please change `1.10.0` to simply `1.10`

File: docs/concepts/configuration/manage-compute-resources-container.md [hunk 2]
```diff
@@ -306,7 +306,7 @@ LastState: map[terminated:map[exitCode:137 reason:OOM Killed startedAt:2015-07-0
 You can see that the Container was terminated because of `reason:OOM Killed`,
 where `OOM` stands for Out Of Memory.
 
-## Local ephemeral storage (alpha feature)
+## Local ephemeral storage (beta feature for version 1.10 and above)
```
Comment (tengqm) [inline docs/concepts/configuration/manage-compute-resources-container.md]: Can we remove the parentheses completely and

- add `{% include feature-state-beta.md %}` in a separate line
- propose this to release-1.10 branch

**Comment by Bradamant3** [comment]
@jingxu97 can you please get the requested changes in asap? Deadline for merging docs PRs for 1.10 is 3/9 (tomorrow US time).
/assign

**Comment by jingxu97** [comment]
@Bradamant3 
I rebased to release 1.10 and had another PR for it https://github.com/kubernetes/website/pull/7685

Is that one ok? Thanks!

**Comment by Bradamant3** [comment]
All's good with #7685, so closing this one unmerged. Thanks, @jingxu97

**Pull request closed without merging**